wordpress-mobile/WordPress-Android
# Remove comments deprecated code

This is a cleanup PR that does the following:

- Extracts needed constants and interfaces to dedicated classes (namely CommentConstants.kt and OnLoadMoreListener.kt)
- Removes deprecated comments list activity, fragments, adapters and related classes.

## To test
- Check CI is green
- Smoke test the comments list (both in My SIte and Notifications, checking no regression was introduced:
   - Check single and batch moderation, 
   - pagination,
   - PTR, 
   - comment details and its behaviours. 
   - Check different filtered lists
   - Try editing comments both in My Site comments and in Notifications
   - Anything else you can think of :) 

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
No unintended area of impact should be there; anyways we are relying on already available unit testing and manual testing.

3. What automated tests I added (or what prevented me from doing so)
This is removing unused classes so no tests addition needed.

PR submission checklist:
